generate_template.php

```diff
@@ -8,7 +8,14 @@
 include('expandFns.php');
 $t = new Template();
 $t->parse_text('{{cite web}}');
-foreach ($_GET as $param=>$value) $t->set($param, $value);
+$count = 0;
```

Comment:
```suggestion
if (count($_GET) > 10) fail_safely() //?
```

Reply:
Can `die` die with some form of error message to the user?
It's not quite clear what's being screened for here.